### ISSUE TITLE
accession numbers truncated?

### ISSUE THREAD

**Issue by jhpoelen** [millerse/US-National-Parasite-Collection#5]
In reviewing https://github.com/jhpoelen/eol-globi-data/issues/307#issuecomment-322831523 , I noticed that [cleaned_up.tsv](../blob/master/cleaned_up.tsv) contains accession ```98139``` for GUID ```E387E263-1EE3-426B-8692-A0FB17D7BA4E```. However, in the original data, accession ```098139.00``` is used. 

It seems that the original accession is interpreted as a number, where it probably should not be.

@millerse @jhammock curious to hear your thoughts on this.

**Comment by jhammock** [comment]
Yes, this has happened to accession numbers before in interfaces that try
to help (excel, etc). Strings like that should be interpreted as text.

Hi, Sarah! If you have access and capacity to look into this, please do. I
trust the new gig is going well :) If you need input about tracking hours
and invoicing while working more intermittently, let's arrange a time to
chat.
